pysmt/fnode.py

```diff
@@ -660,3 +760,8 @@ def _env():
 def _mgr():
     """Aux function to obtain the formula manager."""
     return pysmt.environment.get_env().formula_manager
+
+def _is_bv(node):
+    """Aux function to check if a fnode is a BV."""
+    return (node.is_symbol() and node.symbol_type().is_bv_type()) or \
```

Comment:
To avoid operations that rely on the global environment. Here the node does not know to which env it belongs to. This is shorter (measured as indirection of the call) and independent from the environment.